.github/CODEOWNERS

```diff
@@ -4,9 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/firestore-dpe is the default owner for changes in this repo
-*                       @googleapis/yoshi-java @googleapis/firestore-dpe
-**/*.java               @googleapis/firestore-dpe
+# The @googleapis/cloud-native-db-dpes is the default owner for changes in this repo
+*                       @googleapis/yoshi-java @googleapis/cloud-native-db-dpes
```

Comment:
we should have both teams rather than replace them

Reply:
The `googleapis/firestore-dpe` team doesn't seem to exist 😅  that's why we're replacing it with the `cloud-native-db-dpes` team.

Reply:
oops sorry looked too quickly! That should be replaced with `api-firestore`